docs/_sass/sidebar.scss

```diff
@@ -165,6 +165,10 @@
             text-decoration: none;
             color: #008979;
```

Comment:
@vishalvivekm Instead of using a hardcoded string it would make more sense to use `var(--sidenav-hover)` which will allow us to have consistent color on hover and on active

Reply:
Good call, @yash37158